src/main/java/com/panasi/interview_questions/controller/CategoryController.java

```diff
@@ -50,20 +54,23 @@ public ResponseEntity<CategoryDto> showCategoryById(@PathVariable int id) {
 	}
 	
 	@PostMapping
+	@PreAuthorize("hasRole('USER') or hasRole('ADMIN')")
```

Comment:
for this request better use only admin role, for current time
maybe in future will make some service for getting categories from all users, with aprooving from admin

Reply:
Done

Reply:
> for this request better use only admin role, for current time maybe in future will make some service for getting categories from all users, with aprooving from admin
Done. Also provided admin access for put and delete requests.